harness/attributes/common.yml

```diff
@@ -40,6 +40,10 @@ attributes.default:
     image: bitnami/zookeeper
     version: latest
 
+  kafka-outbox-relay:
+    image: quay.io/inviqa_images/kafka-outbox-relay
+    version: v1
+
```

Comment:
shouldn't we be standardising it in? https://github.com/inviqa/harness-go/blob/master/harness/attributes/docker.yml#L10-L32

Reply:
also to make things like DB_DRIVER configurable

Reply:
if you are asking whether it should be in the docker.yml, then yeah definitely, will move it 👍🏻

Reply:
and prob the env vars too

Reply:
and yeah we probably should move to `app.services`, 👍🏻 , although it will be a breaking change I think they should be standardised like you said, and we are still on `0.x.x` so no BC promise/guarantees provided

Reply:
https://github.com/inviqa/harness-go/blob/master/_twig/docker-compose.yml/service/mysql.yml.twig#L5-L8

Reply:
it's just `services` rather than `app.services`, but for now moving just the kafka-outbox-relay would make sense

Reply:
yeah the env stuff would be nice too 👍🏻 , will sort that out for `kafka-outbox-relay`

I will do a separate PR to move over the `kafka` and `zookeeper` attributes to be under `services` too